### DISCUSSION TITLE
Use period key on number pad for entering timecode

### DISCUSSION THREAD

**Discussion by davidhlawrence** [videovillage/community#92] [Ideas]
Right now, when entering timecode with a number pad, the period key doesn't work for creating separators between time values. You have to type colon or semicolon on the keyboard. It would be much better if the period key worked as a colon key when entering timecode. NLEs do this and Screen should too!

**Comment by gregcotten** [comment]
Hey! I'll look into this, but in the meantime you don't even need to type separators at all! Just type the time code without separators and Screen will parse it correctly

**Comment by davidhlawrence** [comment]
Heya Greg, You're absolutely right! I didn't even think to try that. What can I say? Muscle memory is strong, lol!😉 Thanks for the fast reply and looking into this. Screen is awesome, btw!
